src/test/java/me/moodcat/database/bootstrapper/Bootstrapper.java

```diff
@@ -15,6 +15,7 @@
 import me.moodcat.database.controllers.ChatDAO;
 import me.moodcat.database.controllers.RoomDAO;
 import me.moodcat.database.controllers.SongDAO;
+import me.moodcat.database.embeddables.VAVector;
```

Comment:
It was a required change for the Test app to work with rooms, because the room retrieval triggered NPE's on getting the vector. It was so small that I attached it to this commit.